easybuild/easyblocks/o/openblas.py

```diff
@@ -29,12 +31,46 @@ def configure_step(self):
             default_opts['TARGET'] = 'POWER8'
 
         for key in sorted(default_opts.keys()):
-            for opts_key in ['buildopts', 'installopts']:
+            for opts_key in ['buildopts', 'testopts', 'installopts']:
                 if '%s=' % key not in self.cfg[opts_key]:
                     self.cfg.update(opts_key, "%s='%s'" % (key, default_opts[key]))
 
         self.cfg.update('installopts', 'PREFIX=%s' % self.installdir)
 
+    def build_step(self):
+        """ Custom build step excluding the tests """
+
+        # Equivalent to `make all` without the tests
+        build_parts = ['libs', 'netlib']
+        for buildopt in self.cfg['buildopts'].split():
+            if 'BUILD_RELAPACK' in buildopt and '1' in buildopt:
+                build_parts += ['re_lapack']
+        build_parts += ['shared']
+
+        # Pass CFLAGS through command line to avoid redefinitions (issue xianyi/OpenBLAS#818)
+        cflags = 'CFLAGS'
+        if os.environ[cflags]:
+            self.cfg.update('buildopts', "%s='%s'" % (cflags, os.environ[cflags]))
+            del os.environ[cflags]
+            self.log.info("Environment variable %s unset and passed through command line" % cflags)
+
+        cmd = "%s make %s %s" % (self.cfg['prebuildopts'], ' '.join(build_parts), self.cfg['buildopts'])
```

Comment:
@lexming You accidentally disabled parallel building (`make -j`) here, which is done by default by `ConfigureMake.build_step`... fixed in #1962

Reply:
I totally missed that, thanks for pointing it out